src/libraries/System.Security.Cryptography.ProtectedData/src/System/Security/Cryptography/ProtectedData.cs

```diff
@@ -19,7 +22,148 @@ public static byte[] Protect(byte[] userData, byte[]? optionalEntropy, DataProte
+                outputData: out outputData
+            );
+            Debug.Assert(result);
+            Debug.Assert(outputData != null);
+            return outputData;
+        }
+
+        /// <summary>
+        /// Encrypts the data in a specified byte span and returns a byte array that contains the encrypted data.
+        /// </summary>
+        /// <param name="userData">A byte span that contains data to encrypt.</param>
+        /// <param name="scope">One of the enumeration values that specifies the scope of encryption.</param>
+        /// <param name="optionalEntropy">
+        /// An optional additional byte span used to increase the complexity of the encryption,
+        /// or empty for no additional complexity.
+        /// </param>
+        /// <returns>A byte array representing the encrypted data.</returns>
+        /// <exception cref="CryptographicException">The encryption failed.</exception>
+        /// <exception cref="NotSupportedException">The operating system does not support this method.</exception>
+        /// <exception cref="OutOfMemoryException">The system ran out of memory while encrypting the data.</exception>
+        /// <exception cref="PlatformNotSupportedException">
+        /// Calls to the Protect method are supported on Windows operating systems only.
+        /// </exception>
+        public static byte[] Protect(
+            ReadOnlySpan<byte> userData,
+            DataProtectionScope scope,
+            ReadOnlySpan<byte> optionalEntropy = default)
+        {
+            CheckPlatformSupport();
+
+            byte[]? outputData;
+            bool result = TryProtectOrUnprotect(
+                userData,
+                optionalEntropy,
+                scope,
+                protect: true,
+                allocateArray: true,
+                bytesWritten: out _,
+                outputData: out outputData
+            );
+            Debug.Assert(result);
+            Debug.Assert(outputData != null);
+            return outputData;
+        }
+
+        /// <summary>
+        /// Encrypts the data in a specified byte span and writes  the encrypted data to a byte span.
+        /// </summary>
+        /// <param name="userData">A byte span that contains data to encrypt.</param>
+        /// <param name="scope">One of the enumeration values that specifies the scope of encryption.</param>
+        /// <param name="destination">The buffer to receive the encrypted data.</param>
+        /// <param name="bytesWritten">
+        /// When this method returns, contains the number of bytes
+        /// written to <paramref name="destination"/>.
+        /// </param>
+        /// <param name="optionalEntropy">
+        /// An optional additional byte span used to increase the complexity of the encryption,
+        /// or empty for no additional complexity.
+        /// </param>
+        /// <returns>true if destination was large enough to receive the decrypted data; otherwise, false.</returns>
```

Comment:
```suggestion
        /// <returns>
        /// <see langword="true"/> if <paramref name="destination"/> was large enough to receive the decrypted data;
        /// otherwise, <see langword="false"/>.
        /// </returns>
```

true/false should use see-langword, and destination should use paramref.

Since that makes the line much longer, I broke it up.